src/Display/Tabs/Tabs.tsx

```diff
@@ -10,84 +10,70 @@ import {
   TabsBody,
 } from '../../Style/Display/TabsStyle';
 
-class Tabs extends React.Component<Props, State> {
-  static Pane = TabPane;
+const Tabs: Tabs = ({activeTab, onTabClick, children, className}) => {
+  const [currentIndex, setCurrentIndex] = React.useState(0);
+  const activeTabOrIndex: any = activeTab || currentIndex;
```

Comment:
@franciscrispin try not to use type `any`, in this case you can just use union types